lib/timescaledb/schema_dumper.rb

```diff
@@ -24,6 +20,28 @@ def views(stream)
       super if defined?(super)
     end
 
+    def timescale_hypertables(stream)
+      stream.puts # Insert a blank line above the retention policies, for readability
+
+      @connection.tables.sort.each do |table_name|
+        if Timescaledb::Hypertable.table_exists? &&
+          (hypertable = Timescaledb::Hypertable.find_by(hypertable_name: table_name))
+          timescale_hypertable(hypertable, stream)
+        end
```

Comment:
Probably this block can become a simple navigation in the hypertables, right?

```suggestion
      Timescaledb::Hypertable.find_each do |hypertable|
          timescale_hypertable(hypertable, stream)
      end
```

Reply:
Ooh, I didn't know that method existed! Even better. Accepting.